### ISSUE TITLE
User cannot login

### ISSUE THREAD

**Issue by harrychy8** [microsoftgraph/msgraph-sdk-javascript#797]
# Bug Report

## Description

I tried to log into the app with my newly created account and it did not work. Can someone please take a look?
The account is under a new tenant since with my work account,  it does not have permission to use calendar APIs.

Thank you very much for your time!

**Screenshots:** 
![image](https://user-images.githubusercontent.com/15678482/169180564-9cd50138-00c6-49ac-b419-95141a60696e.png)

**Comment by sebastienlevert** [comment]
Did you use the following tutorial? https://docs.microsoft.com/en-us/graph/tutorials/react 

Adding @jasonjoh for visibility

**Comment by harrychy8** [comment]
Hi guys,

Thank you very much for the fast reply! I have tried the above and here is what I found out:

- From the `upn `and `scp`, I think the user and the permissions are correct
![image](https://user-images.githubusercontent.com/15678482/169355789-62bbe776-288f-45a4-b978-d0ea852a510c.png)

- I was able to log in and use any queries in Graph Explorer.

- Also the app was built by this link https://developer.microsoft.com/en-us/graph/quick-start

Thank you again for the help! Is there anything else that I can try?
